conformance/service_import.go

```diff
@@ -87,6 +97,44 @@ var _ = Describe("", func() {
 				reportNonConformant(fmt.Sprintf("The value %q is not a valid IP", serviceImport.Spec.IPs[0])))
 		})
 
+	Specify("A ServiceImport should only exist as long as there's at least one exporting cluster",
```

Comment:
Maybe we could better emphasize that here we are testing (could be via a code comment not necessarily the `Specify` I guess) the deletion with two clusters exporting a service compared to the previous test?

Reply:
I could remove the unexport part from the previous test but I added it there b/c that test doesn't require two clusters and I think it's good that we try to test as much as we can if only one cluster is specified. The distinction with this test is that it ensures the SI isn't deleted when the first cluster service is unexported.  I think that "_as long as there's at least one exporting cluster_" implies the distinction but, either way, I can add a code comment stating the above.

Reply:
I think both of your suggestions are fine! My main concern was that if you read only the specify you could think that one of the test may be redundant so just slightly clarifying this would be awesome!